js/rendererjs/disabledplugin.js

```diff
@@ -0,0 +1,24 @@
+import React, { PropTypes } from 'react'
+
+const containerStyle = {
+	display: 'flex',
+	alignItems: 'center',
+	justifyContent: 'center',
+	flexDirection: 'column',
+	backgroundColor: '#C6C6C6',
+	width: '100%',
+	height: '100%',
+}
```

Comment:
some inline CSS, eh? Is there a good metric for when to use this and when to define the style in a .css file?